pandas/sparse/tests/test_array.py

```diff
@@ -829,6 +829,51 @@ def test_numpy_mean(self):
         tm.assertRaisesRegexp(ValueError, msg, np.mean,
                               SparseArray(data), out=out)
 
+    def test_abs(self):
```

Comment:
For sanity purposes, include a test where `sparse` has just positive numbers and a positive `fill_value`.